code/game/machinery/Sleeper.dm

```diff
@@ -105,16 +105,17 @@
 				dat += "<HR>No Dialysis Output Beaker is present.<BR><HR>"
 
 			for(var/chemical in connected.available_chemicals)
-				dat += "Inject [connected.available_chemicals[chemical]]: "
+				dat += "Inject [connected.available_chemicals[chemical]]:"
```

Comment:
```suggestion
				dat += "<label style='width:200px'>Inject [connected.available_chemicals[chemical]]:</label>"
```

This will add a fixed width around the inject to align the buttons afterwards. You should test that 200px is actually a good number, but overall this should help align things a little more.

Reply:
![image](https://user-images.githubusercontent.com/45076386/58762046-e8031d80-8510-11e9-9a10-c147a19a6115.png)
put that in to test, doesn't actually seem to do anything.

Reply:
I missed adding `display: inline-block;`

---

It should be
`<label style='width:200px; display: inline-block'>`